src/routes/tools/comparative-slr/_ComparativeSLRMap/Rasters.svelte

```diff
@@ -0,0 +1,169 @@
+<script>
+  import { onDestroy } from "svelte";
+  import { getTileUrl } from "../_data";
+
+  export let map;
+  export let mapStyle;
+  export let dataLayers = [];
+  export let scenario;
+  export let timeFrame;
+
+  // these props only used to determine if raster layers should be updated
+  let curScenario;
+  let curTimeFrame;
+  let styleDataChangeListenerAdded = false;
+
+  const VISIBLE = "visible";
```

Comment:
Constants for strings like this are nice because they prevent you from misspelling them (at least for me they are!)